src/layouts/styles.js

```diff
@@ -2,15 +2,61 @@ import facepaint from 'facepaint'
 const breakpoints = [576, 768, 1024, 1200]
 const mq = facepaint(breakpoints.map(bp => `@media (min-width: ${bp}px)`))
 
-export const styles = {
-  pageWrapper: mq({
-    overflow: 'auto',
-    width: '100%',
-    height: 'auto',
-    display: 'block',
-    position: 'relative',
-    paddingLeft: ['32px', '32px', '80px', '135px', '160px'],
-    paddingRight: ['32px', '32px', '80px', '135px', '160px'],
-    paddingBottom: ['60px', '60px', '60px', '226px'],
-  }),
-}
+export const pageWrapper = mq({
```

Comment:
@Alphascythian if this styles are meant for the project page, then let's create a folder called `Project` inside the pages folder and add the styles there. Use the components as examples